bin/validate.py

```diff
@@ -0,0 +1,63 @@
+from operator import truediv
+from os import listdir, getcwd, path
+import yaml, re
+
+def try_get_value(chart: dict, key: str) -> str:
```

Comment:
`chart.get(key, "")` instead?

Reply:
The Chart Resting lint function is quite opinionated and doesn't check for exact values, just for the existence of values. I am using it in addition to this script, but this lets us easily add our own custom rules.

I should probably rename `fail()` to something more descriptive like `print_failure()`. It did originally also raise an exception but that was less helpful than just printing the error message and calling `exit(1)` later in the loop.